openedx/core/djangoapps/theming/microsite_transition_helpers.py

```diff
@@ -0,0 +1,19 @@
+"""
+Helper functions for changes required to transition Microsite to theming.
+These Helpers will be removed once microsites have fully transitioned into theming.
+"""
+from openedx.core.djangoapps.theming.helpers import get_current_site
+from openedx.core.djangoapps.theming.models import SiteTheme
+
+
+def current_request_has_associated_site_theme():
```

Comment:
Is this method only useful for the transition?  It seems like it could be moved into `openedx.core.djangoapps.theming.helpers` which would avoid the funky import [here](https://github.com/edx/edx-platform/pull/13358/files#diff-395f8f21f3f6cb80d7f9c532b06459f5R55)